### DISCUSSION TITLE
osprogramadores/blog/introducao_a_linguagem_de_programacao_cpp

### DISCUSSION THREAD

**Discussion by giscus** [brenoepics/osprogramadores#1] [General]
# osprogramadores/blog/introducao_a_linguagem_de_programacao_cpp

Tudo que você sempre quis saber sobre Programação de Computadores e nunca teve coragem de perguntar.

http://localhost:5173/osprogramadores/blog/introducao_a_linguagem_de_programacao_cpp

**Comment by brenoepics** [comment]
Hello, world!
